data/mainnet/tokens.js

```diff
@@ -5575,6 +5644,84 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['iFARM', 'DEN-MAR22'],
   },
+  jarvis_2JPY_HODL: {
+    category: VAULT_CATEGORIES_IDS.JARVIS,
+    chain: CHAINS_ID.MATIC_MAINNET,
+    logoUrl: './icons/two-cad.jpg',
+    apyIconUrls: ['./icons/ses.png'],
+    apyTokenSymbols: ['SES'],
+    displayName: 'Jarvis: 2JPY',
+    subLabel: 'HODL',
+    tokenAddress: addresses.MATIC.V2.jarvis_2JPY_HODL.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.jarvis_2JPY_HODL.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.CURVE_POOL,
+      params: [
+        addresses.MATIC.V2.jarvis_2JPY_HODL.Underlying,
+        addresses.MATIC.V2.jarvis_2JPY_HODL.Underlying,
+        18,
+        ['JJPY', 'JPYC'],
+        CHAINS_ID.MATIC_MAINNET,
+      ],
+    },
+    estimateApyFunctions: [
+      {
+        type: 'JARVIS_HODL_V2',
+        params: [
+          0,
+          addresses.MATIC.V2.jarvis_2JPY_HODL.RewardPool,
+          addresses.MATIC.V2.jarvis_2JPY_HODL.Underlying,
+          'jarvis_SES_2JPY',
+          profitSharingCut8Percent,
+        ],
+        extraDailyCompound: false,
+      },
+    ],
+    apyDescriptionOverride: [
+      'Auto harvested <b>SES</b> deposited into <b>SES(MAR22)-2JPY</b> for more APY',
+    ],
+    cmcRewardTokenSymbols: ['iFARM', 'SES'],
+  },
+  jarvis_2CAD_HODL: {
+    category: VAULT_CATEGORIES_IDS.JARVIS,
+    chain: CHAINS_ID.MATIC_MAINNET,
+    logoUrl: './icons/2jpy.png',
+    apyIconUrls: ['./icons/qui.png'],
+    apyTokenSymbols: ['QUI'],
+    displayName: 'Jarvis: 2CAD',
+    subLabel: 'HODL',
+    tokenAddress: addresses.MATIC.V2.jarvis_2CAD_HODL.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.jarvis_2CAD_HODL.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.CURVE_POOL,
+      params: [
+        addresses.MATIC.V2.jarvis_2CAD_HODL.Underlying,
+        addresses.MATIC.V2.jarvis_2CAD_HODL.Underlying,
+        18,
+        ['JCAD', 'CADC'],
+        CHAINS_ID.MATIC_MAINNET,
+      ],
+    },
+    estimateApyFunctions: [
+      {
+        type: 'JARVIS_HODL_V2',
+        params: [
+          0,
+          addresses.MATIC.V2.jarvis_2CAD_HODL.RewardPool,
+          addresses.MATIC.V2.jarvis_2CAD_HODL.Underlying,
+          'jarvis_SES_2JPY',
```

Comment:
Wrong hodl vault

Reply:
> Wrong hodl vault

Could you tell me what's wrong here please?

Reply:
It should be jarvis_QUI_2CAD